draft-irtf-cfrg-frost.md

```diff
@@ -1359,6 +1359,7 @@ The dealer that performs `trusted_dealer_keygen` is trusted to 1) generate good
 
 It is assumed the dealer then sends one secret key share to each of the `NUM_PARTICIPANTS` participants, along with `vss_commitment`.
 After receiving their secret key share and `vss_commitment`, participants MUST abort if they do not have the same view of `vss_commitment`.
+Ensuring a consistent view of this VSS commitment can be achieved by employing a secure broadcast channel.
```

Comment:
```suggestion
The dealer can use a secure broadcast channel to ensure each participant has a consistent view of this commitment.
```

Reply:
I don't get this - the dealer already sends the private key and commitments though a secure channel. What else is needed here? I kinda assumed that participants had to talk between themselves to check if they have the same commitments, but I'm not sure.

Reply:
Exactly, but if they talk amongst themselves to check if they have the same commitments, it has be done in a way that it fulfills the properties of a broadcast channel. Otherwise, some players could cheat and send inconsistent commitments, etc.